doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -1724,9 +1724,55 @@ Ltac2 features a toplevel loop that can be used to evaluate expressions.
    together with its type. This command is pure in the sense that it does not
    modify the state of the proof, and in particular all side-effects are discarded.
 
+Compilation
+-----------
+
+Ltac2 definitions can be compiled to OCaml.  The compiled definitions
+will then run natively instead of using the interpreter.
+
+.. cmd:: Ltac2 Compile {* @qualid }
+
+   Compiles the named Ltac2 definitions and any other Ltac2 definitions they depend on
```

Comment:
FWIW: "Ltac2 definitions" meaning definitions of Ltac2 tactics?  This seems like a poor choice of terminology.  It brings to mind the `Definition` command, which is something completely different, while not using the well-established term "tactic".  (Of course, you're just following the terminology used in the Ltac2 chapter.)